src/data/protocol-registry.ts

```diff
@@ -131,11 +131,20 @@ const protocols: WaylandProtocolRegistryItem[] = [
     },
     {
         id: 'input-method-unstable-v1',
-        name: 'Input method',
+        name: 'Input method v1',
         source: WaylandProtocolSource.WaylandProtocols,
         stability: WaylandProtocolStability.Unstable,
         protocol: require('./protocols/input-method-unstable-v1.json'),
     },
+    {
+        id: 'input-method-unstable-v2',
+        name: 'Input method v2',
+        source: WaylandProtocolSource.WlrProtocols,
```

Comment:
This needs to be changed to `WaylandProtocolSource.External`.

`WaylandProtocolSource.WlrProtocols` is used for protocols found in the [wlr-protocols](https://gitlab.freedesktop.org/wlroots/wlr-protocols) repository, but `input-method-unstable-v2` is actually found in the [wlroots](https://gitlab.freedesktop.org/wlroots/wlroots) repository.

Minor nitpick: Also, can you please move the whole registry entry closer to the bottom of the list where the rest of the `external` protocols are listed.

Reply:
Done

Reply:
I put it with the wlroots one, since it comes with wlroots. For some reason, it's packaged in the `wlroots` repo instead of `wlr-protocols`.